sdk/c/src/ydc-c.c

```diff
@@ -0,0 +1,106 @@
+/**************************************************************
+ * Copyright (c) 2018-2020,Hangzhou Rokid Tech. Co., Ltd.
+ * All rights reserved.
+ *
+ * FileName: ydc-c.c
+ * Description: Dynamic library interface
+ *
+ * Date  :	2019.06.18
+ * Author:  zijiao.wang@rokid.com
+ * Modification: Init version
+ *
+ **************************************************************/
+
+#include <stdlib.h>
+#include <stdint.h>
+#include <unistd.h>
+#include <uv.h>
```

Comment:
I think it's time to support shared libtuv, @legendecas could you please help with this?